### ISSUE TITLE
Перебои работы Интернета в [СПБ (Мегафон, БФА-Телеком)] [14:55 — 16:10]

### ISSUE THREAD

**Issue by nanshakov** [podivilov/runet-blackout#12]
* Медленно прогружается https://dnevnik.ru/
* Провайдер по 2ip - мегафон.

**Comment by nanshakov** [comment]
![изображение](https://user-images.githubusercontent.com/15703204/71357053-4ec35380-2595-11ea-8ec4-e44f6e597196.png)

**Comment by f14a2cd** [comment]
Открывается нормально, если актуально

**Comment by podivilov** [comment]
> Открывается нормально, если актуально

А какой провайдер?

**Comment by f14a2cd** [comment]
> А какой провайдер?
МГТС, Селектел и O2DC
РТК под свердловском тоже ок

**Comment by slomonosov** [comment]
Beeline, РТК, Starlink, Westcall - в МСК нормально
